doc/source/ray-core/ray-dashboard.rst

```diff
@@ -6,74 +6,303 @@ Ray provides a web-based dashboard for monitoring and debugging Ray applications
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/dashboard-pics/profile-button.png
+    :align: center
+
+First, you can download the chrome tracing file by clicking the download button.
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/dashboard-pics/profile_drag.png
+    :align: center
+
+Second, you can use tools like ``chrome://tracing`` or the `Perfetto UI <https://ui.perfetto.dev/>`_ and drop the downloaded chrome tracing file. We will use the Perfetto as it is the recommendation way to visualize chrome tracing files.
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/dashboard-pics/timeline.png
+    :align: center
+
+Now, you can see the timeline visualization of Ray tasks and actors. There are Node rows (hardware) and Worker rows (processes). 
+Each worker rows display a list of events (e.g., task scheduled, task running, input/output deserialization, etc.) happening from that worker over time.
+
+Ray Status
+~~~~~~~~~~
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/dashboard-pics/ray-status.png
+    :align: center
+
+The job page displays the output of the helpful CLI tool ``ray status``, which shows the autoscaler status of the Ray cluster.
+
+The left page shows the autoscaling status, including pending, active, and failed nodes. 
+The right page displays the cluster's demands, which lists demanded resources that cannot be scheduled to the cluster at the moment. This page is useful for debugging resource deadlocks or slow scheduling.
+
+.. note:: 
+
+  The output shows the aggregated information across the cluster (not by a job). If you run more than one job, some of the demands may come from other jobs.
+
+.. _dash-workflow-state-apis:  
+
+Task Table, Actor Table, Placement Group Table
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/dashboard-pics/tables.png
+    :align: center
+
+The dashboard shows a table with the status of the job's tasks, actors, and placement groups. 
+This information is the same as what you would get from the :ref:`Ray state APIs <state-api-overview-ref>`.
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/dashboard-pics/task-table.png
+    :align: center
+
+You can expand the collapse and see a list of each task, actor, and placement group.
+
+.. _dash-node-view:
+
+Cluster View
+------------
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/nodes-view-expand.png
+    :align: center
+
+The cluster view visualizes hierarchical relationship of
+machines (nodes) and workers (processes). Each host consists of many workers, and
+you can see them by clicking the + button. This also shows the assignment of GPU resources to specific actors or tasks.
```

Comment:
We will change "+" icon to arrow icons (> and down arrow) later but we can probably fix it after that.